mw-webapp/src/component/tag/Tag.tsx

```diff
@@ -47,25 +68,34 @@ interface TagProps {
  * WayCard tag component
  */
 export const Tag = (props: TagProps) => {
-  return (
-    <button className={styles.tagContainer}>
-      <span
-        className={clsx(styles.cross, !props.isDeletable && styles.hiddenCross)}
-        onClick={() => {
-          props.isDeletable && props?.onDelete && props.onDelete(props.tagName);
-        }}
-        data-cy={props.cy?.dataCyCross}
-      >
-        X
-      </span>
+  return props.type === TagType.BASIC_TAG ?
+    (
+      <button className={styles.tagContainer}>
+        <span
+          className={clsx(styles.cross, !props.isDeletable && styles.hiddenCross)}
+          onClick={() => {
+            props.isDeletable && props?.onDelete && props.onDelete(props.tagName);
+          }}
+          data-cy={props.cy?.dataCyCross}
+        >
+          X
+        </span>
 
+        <span
+          className={styles.tag}
+          data-cy={props.cy?.dataCyTag}
+        >
+          {props.tagName}
+        </span>
+      </button>
+    )
+    :
+    (
       <span
-        className={styles.tag}
+        className={styles.wayTag}
```

Comment:
let's rename wayTag =>  tag . Brcause this component will be used as skils for users too)

Reply:
I can't rename it to tag, because this wayTag style class contains specific styles different from the basic tag. For example, I can rename it to `cardTag`.

Reply:
Yea, cardTag is okey